bazel/repository_locations.bzl

```diff
@@ -71,8 +71,8 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/google/protobuf/archive/v3.5.0.tar.gz"],
     ),
     envoy_api = dict(
-        commit = "f268484d221509b175c46cce6c4cc3593a0ede5e",
-        remote = "https://github.com/envoyproxy/data-plane-api",
+        commit = "07b79a51679558d6460fd2785b7206d102869f4d",
+        remote = "https://github.com/taiki45/data-plane-api",
```

Comment:
Looks like this snuck in by accident.

Reply:
I'm assuming this was purposeful to align with https://github.com/envoyproxy/data-plane-api/pull/416 to pass CI.  What's the expected workflow for synchronized changes to both repos?

Reply:
Ah, yes, that makes sense. Normally you'd get your change reviewed and merged over at data-plane-api, then update the hash in your PR here (if someone else updates it first past your changes in data-plane-api, just take the newer hash when you merge master).

It's fine to push a PR for review pointing at the author's repo to get early feedback, but obviously it needs to get updated before the PR can be merged. It's probably good practice to comment on your own PR to mark it as needing work.

Reply:
Yes, I read the workflow at https://github.com/envoyproxy/data-plane-api/blob/master/CONTRIBUTING.md#api-changes but leaving the comment encourages reviews. I will do that, thanks!